projects/feature-activation/0005-miner-update-request.md

```diff
@@ -0,0 +1,61 @@
+# Hathor Integration Update Request
+
+## Description
+
+We’re currently in the testing phase of a project called Feature Activation, which is a mechanism used to activate new consensus-agreed features on Hathor Network. In the future, it will be used to release important features such as Nano Contracts and increasing the maximum Merkle Path length, which will allow miners to find more valid merge mined blocks. Here’s a [blog post](https://blog.hathor.network/hathors-feature-activation-process-miners-lead-the-way-d7f43b12978d) about it.
+
+In order for that to work, we need miners to propagate a support signal for features through their mined blocks. This is simply a repurposed byte in the block data. We’ll soon have a complete guide on how to interact with feature signals, but everything should work by default and mostly transparently. However, at this time, we kindly request your cooperation.
+
+It’s likely that you need to update your integration with Hathor to correctly propagate those signals. We have a new field called `signal_bits` in our block template, that you currently use. This field is an integer and represents a single byte. You simply have to set this as the first byte in the Hathor blocks you generate, and everything should work. Before, the `version` field occupied the first 2 bytes. Now, the `signal_bits` is the first byte, and the version field is the second byte.
+
+We of course are available for any questions or help that you might need. Thank you in advance!
+
+## Updating the Integration
+
+In order to support this new feature, a simple change must be made.
+
+### 1. Identifying where the Hathor block is generated
+
+Your current integration most certainly reads our Block template from one of our HTTP or WebSocket endpoints. The fields provided in those endpoints are the following:
+
+```python
+versions: set[int]
+signal_bits: int
+reward: int
+weight: float
+timestamp_now: int
+timestamp_min: int
+timestamp_max: int
+parents: list[bytes]
+parents_any: list[bytes]
+height: int
+score: float
+```
+
+Identify where your integration code reads this data and constructs the block.
+
+### 2. Adding `signal_bits` to the block data
+
+Currently, you integration ignores the `signal_bits` field, which didn't exist before. You likely set the first byte of the block data as `0x00`, or just use the `version` field to set the first two block bytes. You must change this so the `signal_bits` field is the first block byte, and the `version` field is the second block byte.
```

Comment:
you integration -> your integration

Also worth mentioning that every previous version of the full node will ignore the value of the first byte, so using the `signal_bits` will not affect the propagation of blocks to peers that have not updated to the latest version.